test/integration/project_test.go

```diff
@@ -157,6 +157,10 @@ func TestProjectBranches(t *testing.T) {
 
 func createProject(t *testing.T, tags map[string]string, groups []string) (string, string) {
```

Comment:
please delete getProjectNameForTest

Reply:
I made a refactor in **_getProjectNameForTest_**, now it's using **_GenerateRandomProjectNameForScan_**
so now we generate a project name only one time per run, and get it for each test that needs a project name
only test that needs a new project name will call to **_GenerateRandomProjectNameForScan_** and this will write it also to the file as we are doing for the first project that have been created for all tests....